tests/load/locustfiles/locustfile.py

```diff
@@ -131,19 +172,56 @@ def register(self) -> None:
             AssertionError: If the user fails to register a channel
             ValidationError: If the register message schema is not as expected
         """
+        if not self.uaid:
+            raise RescheduleTask()
+
         chid: str = str(uuid.uuid4())
+        control: bool = True
+
+        with self._time_event(name="send_notification") as timer:
+            while control:
+                try:
+                    body = json.dumps(dict(messageType="register", channelID=chid))
+                    self.ws.send(body)
+                    reply = self.ws.recv()
+                except ssl.SSLError:
+                    # if there is an error, disconnect and retry
+                    self.disconnect()
+                    self.connect()
+                    self.hello()
```

Comment:
This is also doing flow-of-control through a try-catch, which should be avoided.

In the case there is an error instead of a retry, we should report the error in the performance metrics.
It could be we need to stop the user after.